next.config.js

```diff
@@ -5,6 +5,30 @@ const withBundleAnalyzer = require('@next/bundle-analyzer')({
 module.exports = withBundleAnalyzer({
   pageExtensions: ['mdx', 'ts', 'tsx'],
   productionBrowserSourceMaps: true,
+  async redirects() {
+    return [
+      {
+        source: '/analysis',
+        destination: '/learn',
+        permanent: false
+      },
+      {
```

Comment:
```suggestion
      },
      {
       source:'/projects/analyst'
       destination:'/learn',
       permanent: false
      },
      {
```
(see https://www.google.com/search?q=%22conveyal.com/projects/%22)